packages/aws-cdk/lib/api/hotswap/ecs-services.ts

```diff
@@ -119,11 +119,11 @@ export async function isHotswappableEcsServiceChange(
 
         // Step 3 - wait for the service deployments triggered in Step 2 to finish
         // configure a custom Waiter
-        (sdk.ecs() as any).api.waiters.deploymentToFinish = {
-          name: 'DeploymentToFinish',
+        (sdk.ecs() as any).api.waiters.deploymentCompleted = {
+          name: 'DeploymentCompleted',
```

Comment:
I'm not well versed in this arena, whats the difference between `deploymentToFinish` and `deploymentCompleted`?

Reply:
It's purely just naming :) I describe this more in my original PR but because we can now detect that the deployment failed I noticed the error message based on this naming wasn't clear what had occured

Before:

```
❌  cdktest-0l5zgdtshw5-ecs-hotswap failed: ResourceNotReady: Resource is not in the state deploymentToFinish
```

After:

```
❌  cdktest-0l5zgdtshw5-ecs-hotswap failed: ResourceNotReady: Resource is not in the state deploymentCompleted
```

Reply:
Ah, I just reminded myself from my commit message for this change as well: the `FAILED` intermediate state for the deployment in the ECS API exists for a reasonably short time. So this change also ensures that we catch that intermediate `FAILED` state